src/storage/filesystem.py

```diff
@@ -1,173 +1,215 @@
             try:
-                remove(path.join(self.baseDir, FilesystemInventory.objectDir, hexlify(hash), FilesystemInventory.metadataFilename))
+                os.remove(os.path.join(workdir, self.metadataFilename))
             except IOError:
                 pass
             try:
-                remove(path.join(self.baseDir, FilesystemInventory.objectDir, hexlify(hash), FilesystemInventory.dataFilename))
+                os.remove(os.path.join(workdir, self.dataFilename))
             except IOError:
                 pass
             try:
-                rmdir(path.join(self.baseDir, FilesystemInventory.objectDir, hexlify(hash)))
+                os.rmdir(workdir)
             except IOError:
                 pass
 
     def __iter__(self):
         elems = []
-        for streamDict in self._inventory.values():
-            elems.extend (streamDict.keys())
+        for streamDict in self._inventory.itervalues():
+            elems.extend(streamDict.keys())
         return elems.__iter__()
 
     def __len__(self):
         retval = 0
-        for streamDict in self._inventory.values():
+        for streamDict in self._inventory.itervalues():
             retval += len(streamDict)
         return retval
 
     def _load(self):
         newInventory = {}
         for hashId in self.object_list():
             try:
-                objectType, streamNumber, expiresTime, tag = self.getMetadata(hashId)
+                objectType, streamNumber, expiresTime, tag = \
+                    self.getMetadata(hashId)
                 try:
-                    newInventory[streamNumber][hashId] = InventoryItem(objectType, streamNumber, None, expiresTime, tag)
+                    newInventory[streamNumber][hashId] = InventoryItem(
+                        objectType, streamNumber, None, expiresTime, tag)
                 except KeyError:
                     newInventory[streamNumber] = {}
-                    newInventory[streamNumber][hashId] = InventoryItem(objectType, streamNumber, None, expiresTime, tag)
+                    newInventory[streamNumber][hashId] = InventoryItem(
+                        objectType, streamNumber, None, expiresTime, tag)
             except KeyError:
-                print "error loading %s" % (hexlify(hashId))
-                pass
+                print("error loading %s" % hexlify(hashId))
         self._inventory = newInventory
-#        for i, v in self._inventory.items():
-#            print "loaded stream: %s, %i items" % (i, len(v))
 
     def stream_list(self):
+        """Return list of streams"""
         return self._inventory.keys()
 
     def object_list(self):
-        return [unhexlify(x) for x in listdir(path.join(self.baseDir, FilesystemInventory.objectDir))]
+        """Return inventory vectors (hashes) from a directory"""
+        return [
+            unhexlify(x) for x in os.listdir(
+                os.path.join(self.baseDir, self.objectDir))]
 
     def getData(self, hashId):
```

Comment:
Am I understand right that `hashId` has the same meaning as `hash` argument above, which I renamed as `invhash`, for example in line 83?